integration_tests/tests/integration/in_cluster.rs

```diff
@@ -333,3 +333,217 @@ async fn in_cluster_fixed_size() {
     )
     .await;
 }
+
+mod rate_limits {
+    #![cfg(feature = "in-cluster-rate-limits")]
```

Comment:
This test module is gated on a non-default feature so that downstream environments that don't have rate limits set up can still run the other integration tests.